bindata/network/ovn-kubernetes/common/001-crd.yaml

```diff
@@ -316,12 +316,12 @@ status:
   conditions: []
   storedVersions: []
 ---
+---
```

Comment:
nit: can you remove this extra separator ?

Reply:
oh woops haha

Reply:
done! thanks for catching that Peri!